apps/examples/lcd_test/example_lcd.c

```diff
@@ -254,13 +254,93 @@ static void test_bit_map(void)
 	free(lcd_data);
 }
 
+static int test_fps_helper(int color){
+	int fd_rtc = 0, fd_lcd = 0, p = 0;
+	char port[20] = { '\0' };
+	size_t len;
+
+	fd_rtc = open("/dev/rtc0", O_RDWR);
+	if (fd_rtc < 0) {
+		printf("ERROR: LCD FPS test, Fail to open rtc.\n");
+		return 0;
+	}
+
+	len = xres * yres * 2 + 1;
+	uint8_t *lcd_data = (uint8_t *)malloc(len);
+	if (lcd_data == NULL) {
+		printf("malloc failed for lcd data : %d\n", len);
+		close(fd_rtc);
+		return 0;
+	}
+	for (int i = 0; i < xres * yres * 2; i += 2) {
+		lcd_data[i] = (color & 0xFF00) >> 8;
+		lcd_data[i + 1] = color & 0x00FF;
+	}
+
+	struct lcddev_area_s area;
+	area.planeno = 0;
+	area.row_start = 0;
+	area.row_end = yres - 1;
+	area.col_start = 0;
+	area.col_end = xres - 1;
+	area.stride = 2 * xres;
+	area.data = lcd_data;
+
+	sprintf(port, LCD_DEV_PATH, p);
+	fd_lcd = open(port, O_RDWR | O_SYNC, 0666);
+	if (fd_lcd < 0) {
+		printf("ERROR: Failed to open lcd port : %s error:%d\n", port, fd_lcd);
+		free(lcd_data);
+		close(fd_rtc);
+		return 0;
+	}
+
+	struct rtc_time start_time = RTC_TIME_INITIALIZER(1970, 1, 1, 0, 0, 0);
+	struct rtc_time end_time = RTC_TIME_INITIALIZER(1970, 1, 1, 0, 0, 0);
+
+	ioctl(fd_rtc, RTC_SET_TIME, (unsigned long)&start_time);
+	ioctl(fd_rtc, RTC_SET_TIME, (unsigned long)&end_time);
```

Comment:
1. As @ewoodev left, no need to set time twice.
2. If the RTC is already running, this example changes the RTC time. I think we can read current time for start and then read again at the end.

Reply:
Yes it was unnecessary, I have removed one of them.

Reply:
2nd is not concerned. If you keep RTC_SET_TIME with 1970-01-01, the system time will be changed by example running. I think it's wrong. I think we don't need to set the time. Just read is enough. Please check it again

Reply:
yes sir, I have tested, and ioctl were unnecessary. i have removed both the ioctl which does the set time.